packages/mobile/src/navigator/DrawerNavigator.tsx

```diff
@@ -242,7 +242,15 @@ export default function DrawerNavigator() {
         <Drawer.Screen
           name={Screens.DAppsExplorerScreen}
           component={DAppsExplorerScreen}
-          options={{ title: t('dappsScreen.title'), drawerIcon: DappsExplorer }}
+          options={{
```

Comment:
Thanks so much!

I think this also affects the Supercharge screen (`ConsumerIncentivesHomeScreen`) and it would be ideal to reload  as well so we check if the user has available Supercharge rewards ready. Would you mind adding that to this PR as well?